scripts/isWorkspace.js

```diff
@@ -1,13 +1,7 @@
 import path from "node:path";
 import {fileURLToPath} from "node:url";
-const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
-// Using CommonsJS require.resolve as long as import.meta.resolve is experimental
-import {createRequire} from "node:module";
-const require = createRequire(import.meta.url);
-
-const cPath = require.resolve("@ui5/cli/package.json");
-const cliPath = path.relative(__dirname, path.dirname(cPath));
+const cliPath = path.relative(__dirname, fileURLToPath(import.meta.resolve("@ui5/cli/package.json")));
```

Comment:
`__dirname` seems to be undefined here. Should it be `import.meta.dirname`?

```suggestion
const cliPath = path.relative(import.meta.dirname, fileURLToPath(import.meta.resolve("@ui5/cli/package.json")));
```

Reply:
True! Thanks